bower.json

```diff
@@ -28,7 +28,7 @@
     "iron-fit-behavior": "PolymerElements/iron-fit-behavior#2.0-preview",
     "neon-animation": "PolymerElements/neon-animation#2.0-preview",
     "paper-styles": "PolymerElements/paper-styles#2.0-preview",
-    "web-animations-js": "web-animations/web-animations-js#^2.2.0"
+    "web-animations-js": "web-animations/web-animations-js#^2.2"
```

Comment:
Yes, it's probably the right place where to put it. Users will be obliged to include this dependency in their bower, which ultimately is a good thing, but for sure source of confusion at the first upgrade. Hopefully the readme will be helpful for them!